pandas/tests/arithmetic/test_numeric.py

```diff
@@ -382,17 +385,12 @@ def test_div_int(self, idx):
         expected = Index(idx.values / 2)
         tm.assert_index_equal(result, expected)
 
-    def test_mul_int(self, idx):
-        result = idx * 1
-        tm.assert_index_equal(result, idx)
+    @pytest.mark.parametrize('op', [operator.mul, ops.rmul, operator.floordiv])
```

Comment:
future nit, might as well import the other ``operator`` to ``ops`` to make things consistent